File: Part4/C++/Function_IncorrectCapitalization.cpp
#include <stdio.h>
#include <stdlib.h>

int main()
{
    char userInput[ 100 ];
    
    printf( "Type 1 and press enter.\n" );
    fgets( userInput, sizeof( userInput ), stdin );
    
    // Function name has wrong capitalization
    int enteredInteger = Atoi( userInput );
    
    printf( "The user entered the integer %d\n", enteredInteger );
    
    return 0;
}

File: Part4/Go/Function_IncorrectCapitalization.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	var userInput string

	fmt.Println( "Type 1 and press enter." )
	fmt.Scanln( &userInput )

	// Function name has wrong capitalization
	enteredInteger, _ := strconv.atoi( userInput )

	fmt.Printf( "The user entered the integer %d\n", enteredInteger )
}
